pyproject.toml

```diff
@@ -46,7 +46,7 @@ rich = "^12.4.4"
 adani = "^1.0.2"
 LeProHQ = "^0.2.5"
 # pineappl dependency
-pineappl = { version = "^0.8.2", optional = true }
+pineappl = { version = "^1.1.0", optional = true }
```

Comment:
minor question: do we need 1.1 or is 1.0 sufficient? (please adjust PR title if necessary)

Reply:
(and PR description :upside_down_face: )